packages/salesforcedx-vscode-core/src/orgBrowser/browser.ts

```diff
@@ -40,8 +41,24 @@ export class OrgBrowser {
     throw this.initError();
   }
 
+  private async getConfigAggregator(): Promise<ConfigAggregator> {
+    const origCurrentWorkingDirectory = process.cwd();
+    const rootWorkspacePath = getRootWorkspacePath();
+    // Change the current working directory to the project path,
+    // so that ConfigAggregator reads the local project values
+    process.chdir(rootWorkspacePath);
+    const configAggregator = await ConfigAggregator.create();
+    // Change the current working directory back to what it was
+    // before returning
+    process.chdir(origCurrentWorkingDirectory);
+    return configAggregator;
+  }
+
   public async init(extensionContext: ExtensionContext) {
-    const username = await OrgAuthInfo.getDefaultUsernameOrAlias(false);
+    const configAggregator = await this.getConfigAggregator();
+    const username: string | undefined = configAggregator.getPropertyValue(
+      OrgConfigProperties.TARGET_ORG
+    );
```

Comment:
just to call it out why'd you have to type the username.  Shouldn't it be able to infer the type?

Reply:
If I remove it typescript complains about passing it to the constructor for this next type metadata outline provider
<img width="1193" alt="2022-08-01_15-02-23" src="https://user-images.githubusercontent.com/46458081/182253556-14bfa785-4771-432a-821f-0c02fc24a4fa.png">
:

Reply:
interesting